_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,130 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] for which one of the following is true:
```

Comment:
Should there be a mention somewhere that the image is not decorative? ("convey information")

Either by checking roles rather than element (can be in applicability), or adding a condition in expectation about "the image is [pure decoration][]".

Reply:
This was one of the the first thoughts that crossed my mind when I started thinking about this rule. I discussed this with @WilcoFiers and he easily convinced me that this is not relevant for this SC. If you are presenting text then you should use text and not images of text unless the presentation is essential. You can always mark text as decorative if it is not being used to convey information.

Reply:
🤔 I have to disagree with the conclusion…
1.4.9, the corresponding AAA criterion, is explicitly allowing images of text that are pure decoration. It would be weird if they were failing 1.4.5…

Reply:
That line in 1.4.9 really made me change my mind. I've updated the applicability.